generators/java/templates/mysqlAndFlyway/pom.xml

```diff
@@ -0,0 +1,99 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId><%= groupId %>
+	</groupId>
+	<artifactId><%= artifactId %>
+	</artifactId>
+	<version>1.0-SNAPSHOT</version>
+
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>1.5.3.RELEASE</version>
+	</parent>
+
+	<description>${project.name} Connector</description>
+
+	<properties>
+		<flyway-core.version>4.0.3</flyway-core.version>
+		<mysql-connector-java.version>5.1.34</mysql-connector-java.version>
+		<service-integration-sdk.version>1.20
```

Comment:
why are this version hardcoded? The generator is getting the version from github to get the latest one